CREDITS.md

```
@@ -79,3 +79,7 @@ File:Sumatra Topography.png
 Stats icon by [Meko](https://thenounproject.com/mekoda/) – https://thenounproject.com/icon/stats-4942475/
 Pay Per Click icon by [Fauzan Adiima](https://thenounproject.com/creator/fauzan94/) – https://thenounproject.com/icon/pay-per-click-2586454/
 Medal icon by [Snow](https://thenounproject.com/snowdoll/) – https://thenounproject.com/icon/medal-4567887/
+
+##Flags
```

markdownlint-cli2 (0.22.0)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)